specs/subresourceintegrity/index.html

```diff
@@ -761,6 +761,12 @@ <h4 id="handling-integrity-violations">Handling integrity violations</h4>
 or execute resources that fail an integrity check, <em>but</em> MUST
 <a href="http://www.w3.org/TR/CSP2/#report-a-violation">report a violation</a>.</p>
 
+    <p>On a failed integrity check, an <code>error</code> event is thrown. Developers
+wishing to provide a canonical fallback resource (e.g. a resource not served
+from a CDN, perhaps from a secondary, slower-to-load source) can catch this
```

Comment:
looks great. maybe "secondary, trusted but slower resource" just to point out why you would do that? I will merge it in after that, I swear :)